### PR TITLE
refactor(Pagination): tailwindify

### PR DESCRIPTION
**Pull request by julio8a**
Supports #1646 #1500

Borders seem to be a continual issue. Had the same on notice. 

I have some `FIXME:` to note and get feedback on. Thx!

From the screener changes, the main change is the brand border bottom. It was 3px, but 3px is not in on scale, so bumped down to 2px, same as alerts and other items that have a brand line on them.

### CODE REVIEW

File: src/components/calcite-pagination/calcite-pagination.scss [hunk 1]
```diff
@@ -1,99 +1,97 @@
...
-    background-color: var(--calcite-ui-foreground-3);
+    @apply bg-foreground-3;
   }
   &.is-disabled {
-    background-color: transparent;
-    pointer-events: none;
+    @apply bg-transparent pointer-events-none;
     & > calcite-icon {
+      //FIXME: There is no tailwind item for this, right?
       opacity: var(--calcite-ui-opacity-disabled);
```
Comment (paulcpederson) [inline src/components/calcite-pagination/calcite-pagination.scss:84]: there is, `opacity-disabled`

File: src/components/calcite-pagination/calcite-pagination.scss [hunk 2]
```diff
@@ -1,99 +1,97 @@
...
+    @apply font-medium text-color-1 border-b-color-brand;
   }
 }
 .previous,
 .next {
   padding: var(--calcite-pagination-spacing);
   &:hover {
+    @apply bg-foreground-2;
+    //FIXME: Does we have a brand color for text?
     color: var(--calcite-ui-brand);
```
Comment (paulcpederson) [inline src/components/calcite-pagination/calcite-pagination.scss:75]: we could use the `link` color here. Probably brand is an accessibility issue when pagination is used on the light gray bg...

Comment (julio8a) [reply src/components/calcite-pagination/calcite-pagination.scss]: @bstifle, ^ this okay?

File: src/components/calcite-pagination/calcite-pagination.scss (hunk 1 above)
Comment (julio8a) [reply src/components/calcite-pagination/calcite-pagination.scss]: updated, thank you!

File: src/components/calcite-pagination/calcite-pagination.scss [hunk 3]
```diff
@@ -1,99 +1,97 @@
...
+    items-center
+    bg-transparent
+    border-none
+    border-opacity-0
+    text-0h
+    text-color-3
+    cursor-pointer
+    font-inherit;
+  border-top: 2px solid transparent;
+  border-bottom: 2px solid transparent;
```
Comment (paulcpederson) [inline src/components/calcite-pagination/calcite-pagination.scss:55]: You could set `border: 0 solid transparent;` and then use `border-b-2 border-t-2` to make it feel more tailwind-like I guess. At least the size is on the tailwind track then

Comment (julio8a) [reply src/components/calcite-pagination/calcite-pagination.scss]: updated

**Review by paulcpederson — APPROVED** [review]

File: src/components/calcite-pagination/calcite-pagination.scss (hunk 3 above)
Comment (paulcpederson) [reply src/components/calcite-pagination/calcite-pagination.scss:55]: @julio8a you have to put the border first _before_ the `@apply`...

Comment (julio8a) [reply src/components/calcite-pagination/calcite-pagination.scss:55]: Tried it and the border disappeared. Can I just leave it like this?

**Review by bstifle — APPROVED** [review]

**Pull request merged**